### DISCUSSION TITLE
Integrating a "Copy as cURL" Option in the App for Easier Request Testing

### DISCUSSION THREAD

**Discussion by casparjones** [darklynx/request-baskets#89] [Ideas]
Will the app include a more detailed "Copy as cURL" option for requests, enabling users to easily import the cURL string into Postman and adjust the host for local testing?

The "Copy as cURL" feature is useful for simplifying request testing and adjusting requests in the console. Both Firefox and Chrome browsers have built-in support for copying requests as cURL strings from their network tools. This feature would be beneficial if integrated into the app, as it would allow users to import the cURL string into Postman or other API testing tools, making it easier to test and adjust requests locally by modifying the host.

Please consider adding this "Copy as cURL" functionality to the app, as it would greatly improve the user experience and streamline the testing process.

**Comment by darklynx** [comment]
I like the idea in general, should not be a complex task to convert a JSON of collected request details into cURL syntax 🤔 

@casparjones if you have any JavaScript library in mind that can generate cURL expressions, please share. Otherwise I'll try to look into it when I have time.
